.github/workflows/staging-daily.yml

```diff
@@ -94,6 +94,15 @@ jobs:
           python -m api.sentry.utils --file ./sentry_rates.csv --project firefox-ios
           ls sentry-slack-firefox-ios.json
         continue-on-error: true
+      - name: Send iOS health notification to Slack
+        id: slack-sentry-ios
+        uses: slackapi/slack-github-action@v2.1.0
+        with:
+          payload-file-path: "./sentry-slack-firefox-ios.json"
+          payload-templated: true 
+          webhook:  ${{ secrets.SLACK_WEBHOOK_URL_TEST_ALERTS_SANDBOX }}
```

Comment:
Sending iOS alerts to sandbox just in case I need something known to work to compare with.